reference/catalog.v3.yml

```diff
@@ -19339,6 +19339,8 @@ definitions:
         type: string
         description: The Manufacturer Part Number (MPN) for the variant.
   productVariant_Full:
+    type: array
```

Comment:
I don't think productVariant_Full is an array. It is an object consisting of productVariant_Base and some additional properties.